include/seqan3/io/sequence_file/input.hpp

```diff
@@ -325,7 +318,7 @@ class sequence_file_input
     //!\brief A seqan3::fields list with the fields selected for the record.
     using selected_field_ids    = selected_field_ids_;
     //!\brief A seqan3::type_list with the possible formats.
-    using valid_formats         = valid_formats_;
+    using valid_formats         = type_list<format_embl, format_fasta, format_fastq, format_genbank, format_sam>;
```

Comment:
But with this change you cannot restrict your sequence file to only accept e.g. FASTA and FASTQ files?

Reply:
This is according to the resolution that we do not yet give the opportunity to specify the set of valid formats. 
This can be added when we resolved how we want to handle formats. It would be already possible to have a dedicated constructor for this, but if not decided otherwise this will first come after 3.1. 
Also please see my notes above. I think valid formats should be removed entirely. First, it won't be needed, second the naming is misleading. Or looking at it in a different way: Is there anything that would make it worthwhile for a file to expose the formats a user can specify? After all, the user can specify any file as long as it fulfills the involved concepts.

Reply:
As I said in the above line, I would still support this for now, but only store it under-the-hood differently :)